integration/ct_integration_test.go

```diff
@@ -202,7 +202,8 @@ func TestCTIntegration(t *testing.T) {
 				t.Errorf("leaf[%d].ts.EntryType=%v; want X509LogEntryType", i, ts.EntryType)
 				continue
 			}
-			// This assumes that the added entries are sequenced in order.
+			// This assumes that the added entries are sequenced in order (which is the case
+			// for Trillian, as each Add[Pre]Chain triggers a distinct batch of leaves).
```

Comment:
This might be true for Trillian with the current mysql storage layer, but it's not required by the storage interface contracts.  Maybe pop a TODO in?